internal/olm/operator/registry/configmap_test.go

```diff
@@ -0,0 +1,70 @@
+package registry
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	apimanifests "github.com/operator-framework/api/pkg/manifests"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-sdk/internal/olm/operator"
+)
+
+var _ = Describe("Configmap", func() {
+
+	Describe("NewConfigMapCatalogCreator", func() {
+		It("should return a configmapcreator with a configuration", func() {
+			cfg := operator.Configuration{
+				Namespace: "testns",
+			}
+
+			ctlog := NewConfigMapCatalogCreator(&cfg)
```

Comment:
create `ctlog` using instance directly. i.e.
```golang
ctlog := &ConfigMapCatalogCreator{
    cfg: INSTANTIATE Configuration HERE
    Package: USE THE ONE YOU HAVE ABOVE
}
```

Reply:
Here Im testing `NewConfigMapCatalogCreator` so I think this should remain as it is

Reply:
Yes, sorry I missed that.